server/sonar-server/src/main/java/org/sonar/server/component/index/ComponentIndexQuery.java

```diff
@@ -35,6 +37,7 @@
 
   private String query;
   private Collection<String> qualifiers = Collections.emptyList();
+  private Set<String> componentUuids = Collections.emptySet();
```

Comment:
Let's discuss that, as I would prefer to fail (not done yet) if an empty set of uuids is given